src/tools/kvstore_tool.h

```diff
@@ -44,6 +44,7 @@ class StoreTool
   StoreTool(const std::string& type,
 	    const std::string& path,
 	    bool need_open_db=true);
+  void start_bluestore(const std::string& path, bool need_open_db);
```

Comment:
could make it a static method which returns `std::unique_ptr<KeyValueDB, Deleter>`.

Reply:
When I try this I get a compile error. Deleter was not declared in this scope.